lib/vagrant-service-manager/plugin.rb

```diff
@@ -8,6 +11,15 @@ class Plugin < Vagrant.plugin('2')
         require_relative 'command'
         Command
       end
+
+      config 'servicemanager' do
+        require_relative 'config'
+        Config
+      end
+
+      action_hook(:servicemanager, :machine_action_up) do |hook|
```

Comment:
Ya @bexelbie . We can omit following lines:

```
config 'servicemanager' do
       require_relative 'config'
       Config
end
```
and respective `config.rb` file since it is not doing much ATM.